poc_config/openmrs/apps/common/formLayout.json

```diff
@@ -612,7 +615,8 @@
                         "id": "fce0e153-215d-422d-985b-d36a3fed687b",
                         "hidden": true,
                         "listenHideEvent": "disable-field-criptococia",
-                        "required":false
+                        "required":false,
```

Comment:
Shouldn't this field be not required by default? I guess this shouldn't be configured here but yes in the form schema in OpenMRS RDE config